help.go

```diff
@@ -65,6 +65,25 @@ type HelpIndenter func(prefix string) string
 // HelpPrinter is used to print context-sensitive help.
 type HelpPrinter func(options HelpOptions, ctx *Context) error
 
+// HelpValueFormatter is used to format help.
+type HelpValueFormatter func(value *Value) string
```

Comment:
"is used to format the help text of flags and positional arguments"

Reply:
Added with the latest commit